### ISSUE TITLE
No simultaneous transitions on an element?

### ISSUE THREAD

**Issue by mblais** [rstacruz/jquery.transit#244]
Simultaneous transitions on a single element don't seem to work if using multiple calls to `$.transit()`.
For example:
````
$w.transit( {opacity: 0.3, queue: false, duration:1500} );
$w.transit( {scale:   0.5, queue: false, duration:1200} );
````
Only `scale` is animated; `opacity` jumps immediately from its current value to 0.3.

Thus it seems impossible with `jquery.transit` to animate two different css properties of an object at the same time, if the transitions have different durations or delays (or if you have other reasons to want to separate the animations into distinct calls to `$.transit()`.

**Comment by ghost** [comment]
use a single $.transit({opacity: 0.3, scale:   0.5}) for Simultaneous transitions

**Comment by mblais** [comment]
Right, but that only allows for a single duration and delay for all the
parameters.
